From f603f72a9ceaaaae25e446da0875ef08036545a3 Mon Sep 17 00:00:00 2001
From: Ioana Ciornei <ioana.ciornei@nxp.com>
Date: Wed, 26 Oct 2016 11:20:26 -0500
Subject: [PATCH 060/107] staging: fsl-mc: whitespace cleanup: align function
 args
Git-commit: 7eef98273d785b3ade4dfdb0b9210cff110e2e79
Patch-mainline: v4.10-rc1
References: bsc#1041022

align function arguments that were not aligned to the starting parenthesis

Signed-off-by: Ioana Ciornei <ioana.ciornei@nxp.com>
Signed-off-by: Stuart Yoder <stuart.yoder@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/bus/dpmcp.h    | 32 ++++++++++++++++----------------
 drivers/staging/fsl-mc/include/dprc.h |  8 ++++----
 2 files changed, 20 insertions(+), 20 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/dpmcp.h b/drivers/staging/fsl-mc/bus/dpmcp.h
index c40253a..0a24c37 100644
--- a/drivers/staging/fsl-mc/bus/dpmcp.h
+++ b/drivers/staging/fsl-mc/bus/dpmcp.h
@@ -61,7 +61,7 @@ struct dpmcp_cfg {
 int dpmcp_create(struct fsl_mc_io	*mc_io,
 		 u32		cmd_flags,
 		 const struct dpmcp_cfg	*cfg,
-		u16		*token);
+		 u16		*token);
 
 int dpmcp_destroy(struct fsl_mc_io *mc_io,
 		  u32 cmd_flags,
@@ -92,45 +92,45 @@ struct dpmcp_irq_cfg {
 int dpmcp_set_irq(struct fsl_mc_io	*mc_io,
 		  u32		cmd_flags,
 		  u16		token,
-		 u8		irq_index,
+		  u8		irq_index,
 		  struct dpmcp_irq_cfg	*irq_cfg);
 
 int dpmcp_get_irq(struct fsl_mc_io	*mc_io,
 		  u32		cmd_flags,
 		  u16		token,
-		 u8		irq_index,
-		 int			*type,
-		 struct dpmcp_irq_cfg	*irq_cfg);
+		  u8		irq_index,
+		  int			*type,
+		  struct dpmcp_irq_cfg	*irq_cfg);
 
 int dpmcp_set_irq_enable(struct fsl_mc_io	*mc_io,
 			 u32		cmd_flags,
 			 u16		token,
-			u8			irq_index,
-			u8			en);
+			 u8			irq_index,
+			 u8			en);
 
 int dpmcp_get_irq_enable(struct fsl_mc_io	*mc_io,
 			 u32		cmd_flags,
 			 u16		token,
-			u8			irq_index,
-			u8			*en);
+			 u8			irq_index,
+			 u8			*en);
 
 int dpmcp_set_irq_mask(struct fsl_mc_io	*mc_io,
 		       u32	cmd_flags,
 		       u16		token,
-		      u8		irq_index,
-		      u32		mask);
+		       u8		irq_index,
+		       u32		mask);
 
 int dpmcp_get_irq_mask(struct fsl_mc_io	*mc_io,
 		       u32	cmd_flags,
 		       u16		token,
-		      u8		irq_index,
-		      u32		*mask);
+		       u8		irq_index,
+		       u32		*mask);
 
 int dpmcp_get_irq_status(struct fsl_mc_io	*mc_io,
 			 u32		cmd_flags,
 			 u16		token,
-			u8			irq_index,
-			u32		*status);
+			 u8			irq_index,
+			 u32		*status);
 
 /**
  * struct dpmcp_attr - Structure representing DPMCP attributes
@@ -153,6 +153,6 @@ struct dpmcp_attr {
 int dpmcp_get_attributes(struct fsl_mc_io	*mc_io,
 			 u32		cmd_flags,
 			 u16		token,
-			struct dpmcp_attr	*attr);
+			 struct dpmcp_attr	*attr);
 
 #endif /* __FSL_DPMCP_H */
diff --git a/drivers/staging/fsl-mc/include/dprc.h b/drivers/staging/fsl-mc/include/dprc.h
index 07a11b3..c40abcd 100644
--- a/drivers/staging/fsl-mc/include/dprc.h
+++ b/drivers/staging/fsl-mc/include/dprc.h
@@ -389,10 +389,10 @@ int dprc_get_obj(struct fsl_mc_io	*mc_io,
 
 int dprc_get_obj_desc(struct fsl_mc_io		*mc_io,
 		      u32		cmd_flags,
-			u16		token,
-			char			*obj_type,
-			int			obj_id,
-			struct dprc_obj_desc	*obj_desc);
+		      u16		token,
+		      char			*obj_type,
+		      int			obj_id,
+		      struct dprc_obj_desc	*obj_desc);
 
 int dprc_set_obj_irq(struct fsl_mc_io		*mc_io,
 		     u32			cmd_flags,
-- 
1.8.5.6

